src/Track.js

```diff
@@ -317,6 +317,31 @@ class Track {
 		return ret
 	}
 
+	/**
+	 * Convert JSON to Track object
+	 * @param {object} json
+	 * @returns {Track}
+	 */
+	fromJSON(json){
```

Comment:
This should also be `static` (which means it should go at the top of the class).